src/Symfony/Component/Security/Core/Util/ClassUtils.php

```diff
@@ -11,14 +11,14 @@
 
 namespace Symfony\Component\Security\Core\Util;
 
-use Doctrine\Common\Util\ClassUtils as DoctrineClassUtils;
+use Symfony\Component\Security\Acl\Util\ClassUtils as AclClassUtils;
+
+@trigger_error('The Symfony\Component\Security\Core\Util\ClassUtils method is deprecated since version 2.8 and will be removed in 3.0. Use Symfony\Component\Security\Acl\Util\ClassUtils instead.', E_USER_DEPRECATED);
```

Comment:
"The ... <s>method</s>class"

And I believe the standard is to do: `'The '.__NAMESPACE__.'\ClassUtils class is ...'`